src/pe/options.rs

```diff
@@ -1,3 +1,5 @@
+pub use crate::options::ParseMode;
```

Comment:
we should reevaluate the `pub` use here and elsewhere; for now `pub use` is required because it's a breaking change otherwise since `options` is a `pub mod`